app/views/notes/index.html.erb

```diff
@@ -1,4 +1,19 @@
 <% notes.each_with_index do |note, i| %>
+
+<div class="row">
+  <div class="card blue-grey file-card col m8 offset-m2">
+    <div class="card-content white-text">
+      <h3 class="card-title">Note <%= i + 1 %></h3>
+        <div>
+          <div>Name: <%= note.note_with_type %></div>
+          <div>Author: <%= note.user_id %></div>
+          <div>Category: <%= note.category.name %></div>
+          <div>Body: <%= note.body %></div>
+        </div>
+    </div>
+  </div>
+</div>
+
```

Comment:
^ this should only be shown if authorized (line 3 through line 15) - it should come after line 19